lib/engine-addon.js

```diff
@@ -8,6 +8,7 @@ var writeFile = require('broccoli-file-creator');
 var Babel = require('broccoli-babel-transpiler');
 var concat = require('broccoli-concat');
 var DependencyFunnel = require('broccoli-dependency-funnel');
+var defaults = require('lodash.defaults');
```

Comment:
we can, but we'd have to bump to 0.6.0 (since 0.5.0 supported 0.12 when it released)